cEP-0004.md

```diff
@@ -35,6 +35,24 @@ The following changes shall be made to the UI:
 
 The below sections describe why and how we should make these changes.
 
+### Mockup
+
+```
+test.py
+|   1| print(•1•)
+|    | [NORMAL] PEP8Bear:
+|    | The code does not comply to PEP8.
+|----|    | /tmp/test.py
+|    |++++| /tmp/test.py
+|   1|    |-print( 1 )
+|    |   1|+print(1)
+|    |
+|    | * Do (n)othing
+|    |   (O)pen file(s)
+|    |   (A)pply the patch
+|    | Choose an action:
```

Comment:
Also we could remove `Choose an action`, and use something like 'Press a button for a specific action` before printing the available actions. Something like:

+|    | Press a button for a specific action:
 +|    | * Do (n)othing
 +|    |   (O)pen file(s)
 +|    |   (A)pply the patch

The user will press the `n` button for `Do (n)othing`
We want this?